iina/Base.lproj/Localizable.strings

```diff
@@ -167,6 +167,12 @@
 "preference.sub_override_level.strip" = "strip";
 "preference.sub_override_level.yes" = "yes";
 
+"preference.sub_override_level.descriptive_text.no" = "No override: render subtitles as specified by the subtitle scripts.";
+"preference.sub_override_level.descriptive_text.force" = "Only allow override using the --sub-ass-* mpv options.";
```

Comment:
This is not what `force` does. This sounds more like what `yes` does. Maybe something like:
`Apply all subtitle styles.`